assets/javascripts/modules/autoComplete.js

```diff
@@ -140,14 +140,16 @@ var updateInput = function (title, value) {
 };
 
 /**
- * Allow auto complete to control and pass over the .value of your suggestion to another optional input
+ * Allow auto complete to control and pass over the .value of your suggestion to one or more optional inputs
  * specified on creation of the autoComplete
+ * if the input is a select, pass the value, otherwise if there is a suggestionDisplayFormat, use that
  * @param value
  */
 var updateTargetInput = function (value) {
-  if ($targetInput) {
-    $targetInput.val(value);
-  }
+  var formattedString = (typeof suggestionDisplayFormat === 'function') ? suggestionDisplayFormat('',value) : value;
+  $.each($targetInputs, function (index, targetEl) {
+    $(targetEl).val($(targetEl).prop('tagName') === "SELECT" ? value : formattedString);
```

Comment:
This feels quite specific and feels like two things are being combined. IMO it would be better to separate out this work. 
Currently you are saying - If the target element is a `select` element then set its value to the value. If not a `select` set target element to the output of the `formattedString`.

Feels like you should be saying:
Set all targetInputs.value to the value
Set all targetDisplayElements to the output of `formattedString`

Which would mean you just need to add another data attribute to accept an element that you would like to display the output of `formattedString `

So then you could leave the `updateTargetInput` as is and create a new method that would `updateDisplayElement` and call that in the same place that `updateTargetInput` is called and send in the `value`

Then you have two things doing one thing well, rather than one thing doing two different things dependant on a hardcoded decision.